packages/errors/src/errors.ts

```diff
@@ -1561,6 +1562,24 @@ export const AllCypressErrors = {
       https://on.cypress.io/configuration
     `
   },
+
+  COMPONENT_TESTING_MISMATCHED_DEPENDENCIES: (dependencies: DependencyToInstall[]) => {
+    const deps = dependencies.map<string>((dep) => {
+      if (dep.detectedVersion) {
+        return `${dep.dependency.type}. Expected ${dep.dependency.minVersion}, found ${dep.detectedVersion}.`
+      }
+
+      return `${dep.dependency.type}. Expected ${dep.dependency.minVersion} but dependency was not found.`
+    })
```

Comment:
Wrapping the variables in a code block looks nicer to me
```ts
return `\`${dep.dependency.type}\`.
```

Reply:
Nice, done. See below post for screenshots.

Reply:
nice